docs/store_learning_materials_in_s3.md

```diff
@@ -0,0 +1,60 @@
+# Storing Learning Materials in and AWS S3 Bucket
+
+By default Ilios will store all of your learning materials in the location
+configured by `file_system_storage_path`. For most environments this is a
+good default. If you wish to run Ilios inside a docker container, in a
+non-traditional hosting environment, or not worry about backing up this data
```

Comment:
should have a comma at end of line.

Reply:
don't want to be a pedantic jerk and hold this up, based on a missing comma. this is good to go!

Reply:
fixed!